Build is failing on Raspberr Pi OS 64bit

[ 70%] Building CXX object CMakeFiles/phd2.dir/calibration_assistant.cpp.o
/home/indi/Projects/phd2/calibration_assistant.cpp: In function ‘wxString VectorToString(const wxString&, const std::vector<wxString>&)’:
/home/indi/Projects/phd2/calibration_assistant.cpp:789:10: error: ‘for_each’ is not a member of ‘std’
  789 |     std::for_each(vec.begin(), vec.end(), [&l](const wxString& s) { l += s.length(); });
      |          ^~~~~~~~
/home/indi/Projects/phd2/calibration_assistant.cpp:792:10: error: ‘for_each’ is not a member of ‘std’
  792 |     std::for_each(vec.begin(), vec.end(),
      |          ^~~~~~~~
make[2]: *** [CMakeFiles/phd2.dir/build.make:1233: CMakeFiles/phd2.dir/calibration_assistant.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:971: CMakeFiles/phd2.dir/all] Error 2
make: *** [Makefile:182: all] Error 2
indi@HOM-INDI-BE-3:~/Projects/build/phd2 $

--- a/calibration_assistant.cpp
+++ b/calibration_assistant.cpp
@@ -37,6 +37,8 @@
 #include "calibration_assistant.h"
 #include "calstep_dialog.h"
 
+#include <algorithm>
+
 enum {defBestDec = 0, defBestOffset = 5, textWrapPoint = 500};
 double const siderealSecsPerSec = 0.9973;
 #define RateX(spd)  (spd * 3600.0 / (15.0 * siderealSecsPerSec))

